xm-commons-domainevent-db/src/main/java/com/icthh/xm/commons/domainevent/db/service/DatabaseFilter.java

```diff
@@ -0,0 +1,14 @@
+package com.icthh.xm.commons.domainevent.db.service;
+
+import com.icthh.xm.commons.domainevent.db.domain.JpaEntityContext;
+import com.icthh.xm.commons.lep.LogicExtensionPoint;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DatabaseFilter {
```

Comment:
@LepService anotation missed, my proposal is to add prefix to store LEPs at the separate folder. For example: @LepService(value = "event.db.filter"